android-components/components/feature/search/src/main/assets/search/list.json

```diff
@@ -362,35 +362,35 @@
     "fr": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "bing", "ddg", "qwant", "wikipedia-fr"
+          "google-b-m", "bing", "ddg", "ebay-fr", "qwant", "wikipedia-fr"
         ]
       }
     },
     "fy-NL": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "ddg", "wikipedia-fy-NL"
+          "google-b-m", "ddg", "ebay-nl", "wikipedia-fy-NL"
```

Comment:
There are some instances like "fy-NL" where the search engine reported in your spreadsheet includes "Bing" for mobile, but doesn't actually appear on the list here. 

For example, fy-NL in your spreadsheet is the following:
['Google', 'Bing', 'Amazon.com', 'Duck Duck Go', 'Wikipedia (fy)']
Desktop: google-b-d, bing, ddg, ebay-nl, wikipedia-fy-NL

I am wondering as a followup if 'Bing' needs to be included here just like how we added it for "uk". I am sure there are other instances where the reported search engine order on mobile differs between the spreadsheet and what we have in `list.json`. This is just one example.